tests/api/utils.py

```diff
@@ -16,6 +18,7 @@ def get_graphql_content(response, *, ignore_errors: bool = False):
     permissions.
     """
     content = _get_graphql_content_from_response(response)
+    flush_post_commit_hooks()
```

Comment:
After running super of `tests.api.conftest.ApiClient.post_graphql` would probably be a better place?

Reply:
Good idea, updated.